project-keeper/src/test/java/com/exasol/projectkeeper/sources/analyze/golang/GoBinaryTest.java

```diff
@@ -7,20 +7,24 @@
 import static org.mockito.Mockito.*;
 
 import java.io.File;
-import java.nio.file.Paths;
 
 import org.hamcrest.Matchers;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 
+@ExtendWith(MockitoExtension.class)
 class GoBinaryTest {
 
     private static final String SEP = File.separator.replace("\\", "\\\\");
-    private static final GoBinary TESTEE = new GoBinary(null, "non-existing-go-binary", null);
+    @Mock
+    private final GoProcess goProcess = mock(GoProcess.class);
```

Comment:
```suggestion
    @Mock
    private GoProcess goProcess;
```
Mock should be created automatically